binder/src/androidTest/java/io/grpc/binder/internal/LeakSafeOneWayBinderTest.java

```diff
@@ -29,28 +30,34 @@
 public final class LeakSafeOneWayBinderTest {
 
   private LeakSafeOneWayBinder binder;
+  private FakeHandler handler = new FakeHandler();
```

Comment:
Nit: `final` since it's never reassigned.

Reply:
done